javascript_nodejs/17.proactive-messages/dialogs/mainDialog/index.js

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const { TurnContext } = require('botbuilder');
+
+class MainDialog {
+    /**
+     * 
+     * @param {Storage} storage A storage system like MemoryStorage used to store information.
+     * @param {BotAdapter} adapter A BotAdapter used to send and receive messages.
+     */
+    constructor (storage, adapter) {
```

Comment:
Is the space between `constructor` and `(` a constructor specific standard? I noticed for the `onTurn` method there is no identical space.

Reply:
good catch